<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>write_data</class>
 <widget class="QDialog" name="write_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Создание медицинской электронной карты</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="AB0_Rh_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Группа крови и резус фактор</string>
   </property>
  </widget>
  <widget class="QLabel" name="SD_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сахарный диабет</string>
   </property>
  </widget>
  <widget class="QLabel" name="AID_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ВИЧ-статус</string>
   </property>
  </widget>
  <widget class="QLabel" name="H_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Гепатит</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AB0_Rh_combo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AB-</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="SD_combo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0 - нет диабета</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>диабет 1-го типа</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>диабет 2-го типа</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="AID_combo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>отрицательный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>положительный</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="H_combo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0 - нет гепатита</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="HZ_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить хроническое заболевание</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="HZ_group">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="HZ_combo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите заболевание:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HZ_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Код заболевания:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HZ_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="HZ_list">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="AS_check">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить активное действующее вещество</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="AS_group">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="AS_combo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите вещество:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Код заболевания:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AS_edit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AS_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="AS_list">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Write_on_card">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>520</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_line">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MN_line">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Surname_line">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
